ui/src/app/app.css

```diff
@@ -24,3 +24,7 @@
   padding-top: 5px;
   padding-bottom: 0px;
 }
+
+.empty-state-text {
+  color: var(--pf-global--Color--200);
+}
```

Comment:
Cool - I didn't know this was possible.  How do I found out what PF colors exist so I can use them in this way?

Reply:
Yeah I learned this for the first time here too :)

https://www.patternfly.org/2020.06/design-guidelines/styles/colors

You can find them here.